src/py/mat3ra/made/tools/utils/coordinate/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .box_coordinate_condition import BoxCoordinateCondition
```

Comment:
It seems that the coordinates and perturbation-related functionality should be outside of `utils` - what exactly do we keep in `utils`, what's the approach?